radis/test/spectrum/test_utils.py

```diff
@@ -118,7 +122,9 @@ def test_perf_profile_from_factory(*args, **kwargs):
     with open("spectrum.prof", "rb") as f:
         assert "calculation_time" in str(f.read())
 
+    os.remove("spectrum.prof")
+
 
 if __name__ == "__main__":
     test_perf_profile()
-    # test_perf_profile_from_factory()
```

Comment:
@erwanp why was it commented out?

Reply:
Just because I was focusing on the other, and forgot to uncommemt. It does not really matter : Pytest picks up all functions starting with test_* . They'll be run whether they're called in the main or not